src/appendTaskCommand.tsx

```diff
@@ -62,29 +63,40 @@ export default function appendTask(props: { arguments: appendTaskArgs }) {
   const tag = noteTag ? noteTag + " " : "";
 
   const selectedVault = vaultName && vaults.find((vault) => vault.name === vaultName);
-  // If there's a configured vault, or only one vault, use that
-  if (selectedVault || vaultsWithPlugin.length == 1) {
+  // If there's a configured vault or only one vault, use that
+  if (selectedVault || vaultsWithPlugin.length === 1) {
     const vaultToUse = selectedVault || vaultsWithPlugin[0];
-    const target = getObsidianTarget({
-      type: ObsidianTargetType.AppendTask,
-      path: notePath,
-      vault: vaultToUse,
-      text: "- [ ] " + tag + content + dateContent,
-      heading: heading,
-      silent: silent,
-    });
-    open(target);
-    popToRoot();
-    closeMainWindow();
+    const openObsidian = async () => {
+      const notePathExpanded = await applyTemplates(notePath);
```

Comment:
I wish this would have made the PR… Still using my dev version because my usecase needs that parsing. Did you have a different idea on how to implement it @KevinBatdorf?

Reply:
I didn't realize it hadn't and not sure how it didn't show up in the diff. I'll open a PR now for it

Reply:
PR is here now: https://github.com/raycast/extensions/pull/7743